giskard/scanner/performance/metrics.py

```diff
@@ -15,6 +15,7 @@ class MetricResult:
     value: float
     affected_samples: int
     raw_values: Optional[np.ndarray] = None
+    ctable_values: Optional[list[list[int]]] = None
```

Comment:
We can't have a contingency table

Reply:
We can be more general and maybe just get a categorical or binary representation?
Something like `binary_counts` or `binary_representation`?

Reply:
Same as above